R/plot.vp.R

```diff
@@ -24,11 +26,25 @@
 #' @examples
 #' data(example_vp)
 #' plot(example_vp)
-#' plot(example_vp, line.col = "blue")
-plot.vp <- function(x, quantity="dens",
+#' plot(example_vp, line_col = "blue")
+plot.vp <- function(x, quantity = "dens",
                     xlab = expression("volume density [#/km"^3*"]"),
-                    ylab = "height [km]", line.col='red', line.lwd=1, ...) {
+                    ylab = "height [km]", line_col = 'red', line.col = 'red',
```

Comment:
line.col and line.lwd still appear as arguments

Reply:
see #126